frontend/src/components/App/Settings/SettingsCluster.tsx

```diff
@@ -147,12 +174,33 @@ export default function SettingsCluster() {
     });
   }
 
+  function storeNewClusterName(name: string) {
+    let actualName = name;
+    if (name === cluster) {
```

Comment:
can you please explain the logic here, Why would we set it to empty name if the cluster name has not changed??

Reply:
Sure. If they are equal, it means that the user hasn't changed the name of the cluster. In this case:
It sets `actualName` to an empty string. This effectively clears the new name, indicating that the user doesn't want to change the cluster name.

It calls `setNewClusterName` to update the state with the empty string, ensuring that the UI reflects this change (if applicable). 

If the user attempts to rename the cluster but provides the same name as before, it indicates that they don't want to change the name. Setting it to an empty string reflects this intention.

Regardless of whether the cluster name has changed or not, it updates the `clusterSettings` state.